### PR TITLE
Feature/gh 623

### PR DESCRIPTION
**Pull request by DmitryBespalov**
# Problem Description
* Synchronization service executes updates (calls to the database) on a background thread even when the phone is locked. At that time, the database is not accessible, so we get an unexpected crash during connection opening. 
* The only thread that does the repeated calls to database is the synchronisation service.
* When app launches, it must refresh token list and account balances (force refresh).
  * This is also needed on "pull-to-refresh" in the tokens overview
  * Also needed when deployment of the safe starts

# Additional information
Before phone gets locked, the app guaranteed goes into background state (checked the behavior on the device, iOS 12.0)

# Solution
Stop the synchronization service when app goes into background. Restart the service when app becomes active again.

### CODE REVIEW

File: MultisigWallet/MultisigWalletDomainModel/SynchronisationService/SynchronisationDomainService.swift [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Foundation
 public protocol SynchronisationDomainService {
 
     func sync()
-    func syncTransactions()
-    func stopSyncTransactions()
+    func start()
```
Comment (sche) [inline MultisigWallet/MultisigWalletDomainModel/SynchronisationService/SynchronisationDomainService.swift]: "sync,start,stop" sounds confusing

Comment (DmitryBespalov) [reply MultisigWallet/MultisigWalletDomainModel/SynchronisationService/SynchronisationDomainService.swift]: please propose new name

File: MultisigWallet/MultisigWalletDomainModel/Wallet/Repeater.swift [hunk 2]
```diff
@@ -10,23 +10,30 @@ public class Repeater {
 
     private let main: (Repeater) throws -> Void
     private let delay: TimeInterval
-    public private(set) var stopped: Bool = false
+    public private(set) var stopped: Bool = true
```
Comment (sche) [inline MultisigWallet/MultisigWalletDomainModel/Wallet/Repeater.swift]: should we create a state enum?

Comment (DmitryBespalov) [reply MultisigWallet/MultisigWalletDomainModel/Wallet/Repeater.swift]: well.. I tried but it seems to make things more complicated.

Comment (DmitryBespalov) [reply MultisigWallet/MultisigWalletDomainModel/Wallet/Repeater.swift]: however, I think it is better

**Pull request merged**